W-14023540 AEH  add instance info to API details page jh (#11)
mulesoft/docs-api-experience-hub: modules/ROOT/pages/viewing-api-details.adoc

The API Details page includes the following information about an API:

> * Name

Reviewer: Maybe these could be bold for UI text?

Author: Yes, good idea. done

+
Name of the API.

* Type
+ 
The API type can be AsyncAPI, HTML, REST, or SOAP.
* Description
+
A description of the API.
* Contact email
+
The email of the contact for the API.
* Version
+
The version of the API.
* Instance
+
The count of instances of the API.
* Client applications
+
The number of client applications of the API.

> The API Console is located in the API Details page. The console component is interactive and shows information about REST APIs and their methods. Endpoints are displayed in alphabetical order. See xref:api-console-specification-and-documentation.adoc[API Console Specification and Documentation] for more information.

Reviewer 2: not only rest api. We can put something like "API specifications"

Author: Yes. I made this change.

== View Instances Information and Copy the URL

> You can view the name and URL of an instance and copy the URL:

Reviewer: ```suggestion
To view the name and URL of an instance and copy the URL:
```
